src/include/stir/listmode/CListModeData.h

```diff
@@ -243,6 +243,7 @@ class CListModeData : public ExamData
   void set_proj_data_info_sptr(shared_ptr<ProjDataInfo>);
   //! Has to be set by the derived class
   //  shared_ptr<ExamInfo> exam_info_sptr;
+  shared_ptr<Scanner> scanner_sptr;
```

Comment:
this has to be removed. It was here before, but is now obsolete (as we have one in `proj_data_info_sptr`